Snippets/Snippets_6/Transports/Throughput/ProvideConfiguration.cs

```diff
@@ -12,5 +13,18 @@ void Configure()
             configuration.LimitMessageProcessingConcurrencyTo(5);
             #endregion
         }
+
+        void ConfigureFromAppConfig()
+        {
+            #region TuningFromAppConfig
+            EndpointConfiguration configuration = new EndpointConfiguration();
+
+            // add your custom key to the appSettings section like this:
+            // <appSettings>
+            //    <add key="MaxConcurrency" value="5"/>
+            // </appSettings>
+            configuration.LimitMessageProcessingConcurrencyTo(int.Parse(ConfigurationManager.AppSettings["MaxConcurrency"]));
```

Comment:
I think giving the sample on how to configure it by reading the settings has more value than showing the config file and we only have a single snippet import on the tuning.md page there. What do you think?